manage.py

```diff
@@ -354,6 +357,8 @@ def frontend_runserver(host, port):
     :param host: host to run dev server on (defaults to 0.0.0.0)
     :param port: defaults to 5000
     """
+    from werkzeug.serving import run_simple
+    from ocd_frontend.wsgi import application
```

Comment:
These lines have to be local since the backend image does not contain the frontend dependencies